velox/exec/HashProbe.cpp

```diff
@@ -1380,11 +1380,24 @@ SelectivityVector HashProbe::evalFilterForNullAwareJoin(
   }
 
   if (buildSideHasNullKeys_) {
+    if (nullKeyProbeHashers_.empty()) {
+      nullKeyProbeHashers_ =
```

Comment:
Thanks for the offline discussion. Maybe put this in prepareNullKeyProbeHashers()?

Reply:
Updated, thanks.